x-pack/metricbeat/module/mssql/_meta/fields.yml

```diff
@@ -0,0 +1,11 @@
+- key: mssql
+  title: "mssql"
+  description: >
+    beta[]
```

Comment:
Use `release: beta` instead of using a static description, and add this entry also to metricset fields file.

Reply:
Done